packages/vms/generate_config_vms_h_curl.com

```diff
@@ -328,8 +328,8 @@ $write cvh "#endif"
 $write cvh "#ifdef USE_CYASSL"
 $write cvh "#undef USE_CYASSL"
 $write cvh "#endif"
-$write cvh "#ifdef USE_DARWINSSL"
-$write cvh "#undef USE_DARWINSSL"
```

Comment:
I think it would make more sense to remove the use of those symbols completely. VMS can't use Secure Transport, there's no point for this #ifdef !